pkg/scheduler/service/transition.go

```diff
@@ -39,8 +39,16 @@ func (t *ClusterStatusTransition) ReconciliationRepository() reconciliation.Repo
 	return t.reconRepo
 }
 
-func (t *ClusterStatusTransition) StartReconciliation(clusterState *cluster.State, preComponents [][]string) error {
+func (t *ClusterStatusTransition) StartReconciliation(runtimeID string, preComponents [][]string) error {
 	dbOp := func() error {
+
+		clusterState, err := t.inventory.GetLatest(runtimeID)
```

Comment:
Please use not `GetLatest`, use `Get` - the reconciliation has to be tracked by config-Id (otherwise it could be that you overwrite the status for a cluster which wasn't yet reconciled.